app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgGraphBuilder.java

```diff
@@ -183,27 +183,41 @@ private void addBgReadingValues() {
         }
     }
 
-    public Line highLine() {
+    public Line highLine(){ return highLine(true);}
+
+    public Line highLine(boolean show) {
         List<PointValue> highLineValues = new ArrayList<PointValue>();
         highLineValues.add(new PointValue((float) start_time, (float) highMark));
         highLineValues.add(new PointValue((float) end_time, (float) highMark));
         Line highLine = new Line(highLineValues);
         highLine.setHasPoints(false);
         highLine.setStrokeWidth(1);
-        highLine.setColor(ChartUtils.COLOR_ORANGE);
+        if(show) {
+            highLine.setColor(ChartUtils.COLOR_ORANGE);
+        } else {
+            highLine.setColor(Color.TRANSPARENT);
+        }
         return highLine;
     }
 
-    public Line lowLine() {
+    public Line lowLine(){ return lowLine(true, false);}
+
```

Comment:
Using multiple booleans is an anti-pattern.  It is better to add a static final const value named something like LINE_VISIBLE or LINE_HIDDEN to make calling code more readable.